boards/mips-malta/malta.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2016, Imagination Technologies Limited and/or its
+ *                 affiliated group companies.
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+
+#include <stdio.h>
+
+#include "periph/uart.h"
+
+#define MIPS_MALTA_ADDR 0xbf000500
+#define MIPS_MALTA_VAL_RST 0x42
```

Comment:
Please can you be consistent in your recommendations. Or even add this to the style guide as a concrete rule. Previously you said you can use 'all parens' or no parens as along as it was consistent within the file. If you want me to use 'all parens' _all_ the time please state that.